packages/block-library/src/embed/index.js

```diff
@@ -331,6 +398,10 @@ const embedAttributes = {
 	providerNameSlug: {
 		type: 'string',
 	},
+	allowResponsive: {
```

Comment:
As mentioned, I think we should use a more specific name for this like `allowVideoResizing`.